app-react/src/jsMain/kotlin/com/ustadmobile/mui/components/Sidebar.kt

```diff
@@ -31,6 +33,7 @@ data class RootScreen(
 val ROOT_SCREENS = listOf(
     RootScreen(ClazzListViewModel.DEST_NAME_HOME, MR.strings.courses, School),
```

Comment:
When importing material icons on React, we should use something like:

```
import mui.icons.material.School as SchoolIcon
```

Because if Android Studio automatically converts this to import mui.icons.material.* it will make Android Studio run extremely slow or crash.